lib/grape/dsl/parameters.rb

```diff
@@ -193,21 +193,25 @@ def declared_param?(param)
 
       alias group requires
 
+      def map_params(params, element)
+        result = if params.is_a?(Array)
```

Comment:
you don't need to assign result here, just leave the `if` block it returns all the right things.